src/AnalysisPrograms/Recognizers/PteropusSpecies.cs

```diff
@@ -0,0 +1,450 @@
+            // Note: if you lower the dB threshold, you need to increase maxDurationSeconds
+            int minHz = profile.GetIntOrNull(AnalysisKeys.MinHz) ?? 800;
+            int maxHz = profile.GetIntOrNull(AnalysisKeys.MaxHz) ?? 8000;
+            double minDurationSeconds = profile.GetDoubleOrNull(AnalysisKeys.MinDuration) ?? 0.15;
+            double maxDurationSeconds = profile.GetDoubleOrNull(AnalysisKeys.MaxDuration) ?? 0.5;
+            double decibelThreshold = profile.GetDoubleOrNull(AnalysisKeys.DecibelThreshold) ?? 9.0;
+
+            var minTimeSpan = TimeSpan.FromSeconds(minDurationSeconds);
+            var maxTimeSpan = TimeSpan.FromSeconds(maxDurationSeconds);
+
+            //######################
+            //2.Convert each segment to a spectrogram. Don't use samples in this recogniser.
+            //var samples = audioRecording.WavReader.Samples;
+
+            // make a spectrogram
+            var sonoConfig = new SonogramConfig
+            {
+                WindowSize = 512,
+                NoiseReductionType = NoiseReductionType.Standard,
+                NoiseReductionParameter = configuration.GetDoubleOrNull(AnalysisKeys.NoiseBgThreshold) ?? 0.0,
+            };
+            sonoConfig.WindowOverlap = 0.0;
+
+            // now construct the standard decibel spectrogram WITH noise removal, and look for LimConvex
+            // get frame parameters for the analysis
+            var sonogram = (BaseSonogram)new SpectrogramStandard(sonoConfig, audioRecording.WavReader);
+            var decibelArray = SNR.CalculateFreqBandAvIntensity(sonogram.Data, minHz, maxHz, sonogram.NyquistFrequency);
+
+            // prepare plots
+            double intensityNormalisationMax = 3 * decibelThreshold;
+            var eventThreshold = decibelThreshold / intensityNormalisationMax;
+            var normalisedIntensityArray = DataTools.NormaliseInZeroOne(decibelArray, 0, intensityNormalisationMax);
+            var plot = new Plot(speciesName + " Territory", normalisedIntensityArray, eventThreshold);
+            var plots = new List<Plot> { plot };
+
+            //iii: CONVERT decibel SCORES TO ACOUSTIC EVENTS
+            var acousticEvents = AcousticEvent.GetEventsAroundMaxima(
+                decibelArray,
+                segmentStartOffset,
+                minHz,
+                maxHz,
+                decibelThreshold,
+                minTimeSpan,
+                maxTimeSpan,
+                sonogram.FramesPerSecond,
+                sonogram.FBinWidth
+                );
+
+            // ######################################################################
+            acousticEvents.ForEach(ae =>
+            {
+                ae.SpeciesName = speciesName;
+                ae.SegmentDurationSeconds = audioRecording.Duration.TotalSeconds;
+                ae.SegmentStartSeconds = segmentStartOffset.TotalSeconds;
+                ae.Name = abbreviatedSpeciesName;
```

Comment:
should probably concatenate the profile name here so it's clear which event was detected.

should also set the `Profile` property on the acoustic event object